conanfile.py

```diff
@@ -0,0 +1,44 @@
+from conans import ConanFile, CMake, tools
+
+class DbPostgreSQLAdapterConan(ConanFile):
+    name = "DbPostgreSQLAdapter"
+    description = "Implementation of C++ DB adapter based on PostgreSQL"
+    url = "https://github.com/systelab/cpp-postgresql-db-adapter"
+    homepage = "https://github.com/systelab/cpp-postgresql-db-adapter"
+    author = "CSW <csw@werfen.com>"
+    topics = ("conan", "db", "sql", "postgreSQL", "adapter")
+    license = "MIT"
+    generators = "cmake_find_package"
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = "*", "!*.yml", "!*.md", "!.gitattributes", "!.gitignore", "!LICENSE"
```

Comment:
Probably you should export just what is inside **src**, not all.

First of all we don't want to export in the package the test project, but also as part of your build with cmake you would be creating new folders at root level that will mess your packacking if not filtered out.

Secondly, unless you want to provide the means to build the library in case a matching package is not found, there is no need to export the sources. Check documentation: https://docs.conan.io/1/reference/conanfile/attributes.html#exports-sources